bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -2903,6 +3007,10 @@ private void fireXAxisSelection(final boolean showTimeOnXAxis) {
       }
    }
 
+   public LinkedHashMap<String, Boolean> get_state_CustomTracksToolBarChart() {
```

Comment:
Can you use ProperCase for names and not use a separator between words

`get_state_CustomTracksToolBarChart` -> `getState_CustomTracksToolBarChart`